src/components/Greeting.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect, useState } from 'react';
+
+function Greeting() {
+  const [randomGreeting, setRandomGreeting] = useState('');
+
+  const fetchRandomGreeting = async () => {
+    try {
+      const response = await fetch('http://localhost:3000/random_greeting');
+      const data = await response.json();
+      setRandomGreeting(data.greeting);
+    } catch (error) {
+      console.log(error);
+    }
+  };
```

Comment:
- According to the project requirements, your project should have a store an action and a reducer. Please make sure your app is connected to redux. Please fix this 🙏